src/messaging/message_bus_base.ts

```diff
@@ -24,12 +54,29 @@ export abstract class MessageBusBase extends MessageBus {
 			async (db) => {
 				const topic: Topic = await db.getTopic(executionContext, { ingressId });
 				const ingress: Ingress = await db.getIngress(executionContext, { ingressId });
-				await db.createMessage(
+
+				const createdMessage: Message = await db.createMessage(
 					executionContext, ingressId,
 					message.messageId, message.headers,
 					message.mediaType, message.ingressBody,
 					message.body
 				);
+				const labelHandlers = this.labelHandlers.get(topic.topicId.uuid);
+				const labelValues: Array<string> = [];
+
+				if (labelHandlers) {
+					const settedLabels = (await Promise.all(labelHandlers.map(e => e.execute(executionContext, createdMessage)))).flat();
```

Comment:
Along with `Promise.all()` I would like to handle possible exceptions as `FExceptionAggregate`

Reply:
fix